<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
	<arg name="model_file" 	default="/srv/data/gait1992/residual_reduction_algorithm/model_adjusted.osim" />

        <node pkg="osrt_ros" type="id_so_jr" 	name="id_node" output="screen">
                <param name="model_file" 		value="$(arg model_file)" />
		<!-- param name="resample_period" 		value="0.01" / -->
		<remap from="/id_node/input_filtered" to="/ik/output_filtered"/>
		<remap from="/in_labels" to="/ik/outlabels"/>
		<!--remap from="/id_node/input2" 	to="/inverse_kinematics_from_file/r_data2"/>
		<remap from="/in_labels2" 	to="/inverse_kinematics_from_file/out_labels"/ -->
		<remap from="/id_node/input2" 	to="/grf_node/output"/>
		<remap from="/in_labels2" 	to="/grf_node/out_labels"/>
        </node>

</launch>

